pkgs/tools/admin/lxd/default.nix

```diff
@@ -14,6 +15,38 @@ let
     [ iptables-nftables-compat ]
   else
     [ iptables ebtables ];
+  lxd-agent = buildGoPackage rec {
+    pname = "lxd-agent";
+    version = "4.8";
+
+    goPackagePath = "github.com/lxc/lxd";
+
+    buildFlags = [ "-ldflags=-extldflags=-static" "-tags libsqlite3" ];
+
+    src = fetchurl {
+      url = "https://github.com/lxc/lxd/releases/download/lxd-${version}/lxd-${version}.tar.gz";
```

Comment:
Just a question: Whats the difference with those compared to the github tarball?